### PR TITLE
Vfb40 - Connect Search and Query Builder to SOLR and Query Servers

### PR DESCRIPTION
**Pull request by jrmartin**

### CODE REVIEW

File: applications/virtual-fly-brain/client/src/shared/subHeader/NarrowSearchFilter/index.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+          columnGap: 1,
+          flexWrap: 'wrap',
+          rowGap: 0.5
+        }}>
+          {tags?.map((tag, index) => (
+            <Chip
+              className="secondary"
+              key={tag + index}
+              sx={{
+                lineHeight: '140%',
```
Comment (vidhya-metacell) [inline applications/virtual-fly-brain/client/src/shared/subHeader/NarrowSearchFilter/index.js:54]: @enicolasgomez please remove lineheight and fontsize

File: applications/virtual-fly-brain/client/src/shared/subHeader/SearchResult/index.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                px: 1
+              }}>
+                {option?.label}
+              </Typography>
+
+              <Box sx={{
+                ml: 'auto',
+                display: 'flex',
+                alignItems: 'center',
+                width: '30%',
```
Comment (vidhya-metacell) [inline applications/virtual-fly-brain/client/src/shared/subHeader/SearchResult/index.js:77]: @enicolasgomez please remove the width from here

Comment (jrmartin) [reply applications/virtual-fly-brain/client/src/shared/subHeader/SearchResult/index.js:77]: @vidhya-metacell If we don't control the width of this element, the Tags overflow the full row displaying results. 

The Search results are coming back with a lot of Tags, more than can be displayed at once.
image

**Comment by jrmartin** [comment]
PR #42 contains this branch now

**Pull request closed without merging**